tests/std/tests/P1614R2_spaceship/env.lst

```diff
@@ -0,0 +1,7 @@
+# Copyright (c) Microsoft Corporation.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+

```

Comment:
This new test needs to be mentioned in `test.lst`, or it won't run in the MSVC-internal repo.